ext/spl/tests/ArrayObject_overloaded_object_incompatible.phpt

```diff
@@ -5,15 +5,15 @@ Objects with overloaded get_properties are incompatible with ArrayObject
 
 $ao = new ArrayObject([1, 2, 3]);
 try {
-    $ao->exchangeArray(new SplFixedArray);
+    $ao->exchangeArray(new DateInterval('P1D'));
```

Comment:
Why is this necessary?

Reply:
> Overloaded object of type DateInterval is not compatible with ArrayObject

ArrayObject checks if get_properties_handler is overloaded and throws if it is.

After my change to spl_fixedarray.c, the get_properties handler of SplFixedArray is no longer overloaded. 

So I changed to DateInterval, which continues to overload the get_properties handler